demo/inferenceUI/README.md

```diff
@@ -0,0 +1,50 @@
+## KAITO InferenceUI Demo
+
+The KAITO InferenceUI Demo provides a sample front-end application that demonstrates
+how to interface with the KAITO Workspace for inference tasks. 
+This guide covers deploying the front-end as a Helm chart in a Kubernetes environment 
+as well as how to run the Python application independently.
+
+### Prerequisites
+
+- Kubernetes cluster with Helm installed (if using helm chart)
+- Python environment (if running the Python file directly)
+- Access to the KAITO Workspace Service endpoint
+
+
+## Deploying with Helm
+To deploy the KAITO InferenceUI Demo using Helm, set the `workspaceServiceURL` environment variable to your 
+Workspace Service  endpoint.
+
+### Configuring the Workspace Service URL
+- Using the --set flag:
+
+```
+helm install inference-frontend ./charts/DemoUI/inference --set env.workspaceServiceURL="http://<CLUSTER_IP>:80/chat"
+```
+ - Using a custom `values.override.yaml` file:
+   ```
+    env:
+        workspaceServiceURL: "http://<CLUSTER_IP>:80/chat"
+   ```
+   Then deploy with custom values file:
+    ```
+    helm install inference-frontend ./charts/DemoUI/inference -f values.override.yaml
+    ```
+
+Replace `<CLUSTER_IP>` with the IP address of your Kubernetes cluster. 
```

Comment:
"the IP address of your Kubernetes cluster." ?    How can user get it?

Reply:
added section